playground/src/main.tsx

```diff
@@ -310,7 +304,7 @@ function registerMonacoLanguageServiceProviders(
         }),
       };
     },
-    triggerCharacters: ["@"], // for attribute completions
+    triggerCharacters: ["@", "."],
```

Comment:
Should we add a note that we need to keep this list in sync with the one in `extension.ts`?

Reply:
Done.